### PR TITLE
Selectors tests

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: src/components/Tasks/TaskItem/container.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ const mapStateToProps = (state, ownProps) => {
   };
 
   return {
-    lastTaskOrder: lastTaskOrder - 1,
+    lastTaskOrder: lastTaskOrder,
```
Comment (plzen) [inline src/components/Tasks/TaskItem/container.js]: Just use `{ lastTaskOrder, ... }`

File: src/store/comments/__tests__/selectors.spec.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import * as selectors from '../selectors';
+import state from '../__mocks__/commentsState';
+
+describe('Comments selectors', () => {
+  const { comments } = state;
+  const { tasks } = state;
+  const taskID = 'lmrhS5Vc9nCOHcGwlSZJ';
+
+  it('getComments()', () => {
```
Comment (plzen) [inline src/store/comments/__tests__/selectors.spec.js]:
```
describe('session selectors', () => {
  describe('getUser', () => {
    const user = {
      id: 1,
    };
    const state = {
      session: {
        user,
      },
    };

    it('returns user', () => {
      expect(getUser(state)).toEqual(user);
    });
  });
});
```

File: src/components/Tasks/TaskItem/container.js (hunk 1 above)
Comment (eugene-chr) [reply src/components/Tasks/TaskItem/container.js]: done

File: src/store/comments/__tests__/selectors.spec.js (hunk 2 above)
Comment (eugene-chr) [reply src/store/comments/__tests__/selectors.spec.js]: done

**Pull request merged**